lib/alchemy/config.rb

```diff
@@ -48,7 +48,8 @@ def env_specific_config
       # If it does not exist, or its empty, it returns an empty Hash.
       #
       def read_file(file)
-        return YAML.load_file(file) || {} if File.exist?(file) # YAML.load_file returns false if file is empty.
+        YAML.safe_load(ERB.new(File.read(file)).result, YAML_WHITELIST_CLASSES, [], true) || {}
+      rescue Errno::ENOENT
```

Comment:
We should not rescue these errors. This will be hard to track down if errors happen, because an empty config doesn't mean something is wrong with the format of that file, right? We want this to be raised as real error so the developer knows were the cause of the problems is.

Reply:
I Agree. I will take it out, but I will keep the original behaviour and we can change it when we implement a service class.

Reply:
Once you take this `rescue` statement out, I believe this is ready to go.